packages/service/ni_measurement_plugin_sdk_service/_internal/grpc_servicer.py

```diff
@@ -181,6 +182,8 @@ def __init__(
         self._measure_function = measure_function
         self._owner = weakref.ref(owner) if owner is not None else None  # avoid reference cycle
         self._service_info = service_info
+        self._configuration_parameters_message_type = service_info.service_class + ".Configurations"
```

Comment:
I'm ok with fixing this assuming that the user impact is limited to emitting warnings in some cases.